luminoth/eval.py

```diff
@@ -44,9 +44,8 @@ def evaluate(dataset_split, config_files, job_dir, watch,
     )
 
     config.train.job_dir = job_dir or config.train.job_dir
-    # Only activate debug for image visualizations.
-    config.train.debug = image_vis
-
+    # Only activate debug for if needed for debug visualization mode.
+    config.train.debug = image_vis == 'debug'
```

Comment:
What if `debug` is turned on and `image_vis` is not debug?

Reply:
This wasn't an option in the previous implementation, but I think the next step will be to change the flags to the config file like in train, when that happens I think it will be something to consider